R/cdisc_data.R

```diff
@@ -7,9 +7,9 @@
 #' for given datasets whose names match ADAM datasets names.
 #'
 #' @inheritParams teal_data
-#' @param join_keys (`join_keys`) object or a single (`join_key_set`) object
+#' @param join_keys (`join_keys`) optional, object or a single (`join_key_set`) object
 #'
-#'   (optional) object with datasets column names used for joining.
+#'   object with datasets column names used for joining.
```

Comment:
Hey, this does not have sense. You just moved `optional` after the first appearance of any class name?
The meaning of this description was that this is either `join_keys` or either `single join_key_set` and in both cases this is optional.
Right now it looks like `join_keys` is optional, but `single join_key_set` is no longer optional.

I would recommend this to be like below:

```r
@param join_keys (`join_keys` or single `join_key_set`) optional object with datasets column names used for joining
```

Reply:
Oh , misread it. thanks. fixed it as per suggestion.